src/test/java/jUnit/JsonParserTest.java

```diff
@@ -12,27 +11,35 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static jUnit.Utils.Constants.FILE_NAME_LIST;
 import static org.junit.jupiter.api.Assertions.*;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
 
-    static Cart initialCart;
-    static JsonParser parser;
+    Cart initialCart;
+    JsonParser parser;
+
+    List<String> fileNameList = Stream.of(
+            "vadim-cart",
+            "vadim1-cart",
+            "vadim2-cart")
+            .collect(Collectors.toList());
 
     @BeforeAll
-    static void setUp() {
+    void setUp() {
         parser = new JsonParser();
-        initialCart = Utils.initCart(0);
     }
 
     @ParameterizedTest
-    @ValueSource(ints = {0})
+    @ValueSource(ints = {0, 1, 2})
     void writeToFileTest(int i) {
+        initialCart = new Cart(fileNameList.get(i));
         parser.writeToFile(initialCart);
-        Cart actualCart = parser.readFromFile(new File("src/main/resources/" + FILE_NAME_LIST.get(i) + ".json"));
+        Cart actualCart = parser.readFromFile(new File("src/main/resources/" + fileNameList.get(i) + ".json"));
         assertAll(() -> assertEquals(initialCart.getCartName(), actualCart.getCartName()),
                 () -> assertEquals(initialCart.getTotalPrice(), actualCart.getTotalPrice()));
     }
```

Comment:
readFromFile test might be improved - you may want to create virtual item, real item, add both to cart and write that info to file. Then only verify total price with tax and cart name is correct.

This change is required as eugen-cart might get changed and that, in turn, will break down the test. To avoid that let's create it inside the test case and then verify if write-read works as expected. Plus you will avoid using magic numbers and strings inside test case